### PR TITLE
Fix import error handling and cleanup participant import icons

### PR DESCRIPTION
**Pull request by luisa-beerboom**
In this PR I did the following:

1. Previously when the import failed with a non-preview-related error (i.e. a string error that doesn't update the preview with an explanation), a snackbar with the error would be shown, but the view would continue showing the preview in a context that implied the import to still be ongoing. -> Fixed that by making the view revert to a clean pre-import state
2. The participant import has a system where trying to change fields that the user has no right to edit will get the field payload removed. This is signified in the preview through a new field state `remove`. Until now that state had no associated icon, causing such fields to show the default icon `block`, which suggests that there is an error on that field, when really there isn't -> Made OpenSlides use the `remove` icon for fields with the `remove` state.

### CODE REVIEW

File: client/src/app/gateways/error-mapping/error-map-utils.ts [hunk 1]
```diff
@@ -54,6 +54,9 @@ const MeetingCreateErrorMap: ErrorMap = new ErrorMap([
  */
 const getActionErrorMap: (data: any) => ErrorMap | null = data => {
     const actionName = Array.isArray(data) && typeof data[0] === `object` ? data[0][`action`] : null;
+    if (typeof actionName === `string` && actionName.endsWith(`.import`)) {
+        return MatchAllErrorMap;
+    }
```
Comment (bastianjoel) [inline client/src/app/gateways/error-mapping/error-map-utils.ts:59]: Move this into the default case.

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/gateways/error-mapping/error-map-utils.ts (hunk 1 above)
Comment (luisa-beerboom) [reply client/src/app/gateways/error-mapping/error-map-utils.ts]: Done

**Review by Elblinator — CHANGES_REQUESTED** [review]
after running into the fix for 1 once you need to reload the page to import the same file again

**Review by Elblinator — APPROVED** [review]
function
Change request from before will become it's own issue

**Pull request merged**